source/gameengine/Physics/Bullet/CcdPhysicsController.cpp

```diff
@@ -1600,6 +1600,9 @@ PHY_IPhysicsController *CcdPhysicsController::GetReplicaForSensors()
 
 			default:
```

Comment:
Well, i doubt that there's a real error here:

The m_collisionShape could be leaked by the return nullptr only if this one was possible when the if (m_shapeInfo) succeed. But this is not the case as the return is in the switch in the else if.

If `cinfo.m_collisionShape = m_shapeInfo->CreateBulletShape(m_cci.m_margin, m_cci.m_bGimpact, !m_cci.m_bSoft);` is called then `return nullptr` is never called and the memory should not leak.

I hope i'm right……